czkawka_gui/i18n/en/czkawka_gui.ftl

```diff
@@ -182,6 +182,11 @@ popover_custom_regex_check_button_entry_tooltip =
 
         This uses the default Rust regex implementation. You can read more about it here: https://docs.rs/regex.
 
+popover_custom_ignore_size_check_button_tooltip =
+        Ignore size of letters.
```

Comment:
I think it should be "case" instead of "size": https://en.wikipedia.org/wiki/Case_sensitivity